dev-qt/qtbase/qtbase-6.5.1-r1.ebuild

```diff
@@ -170,6 +169,7 @@ src_configure() {
 		$(qt_feature gssapi)
 		$(qt_feature libproxy)
 		$(qt_feature sctp)
+		$(qt_feature ssl openssl)
```

Comment:
This is going to be overwritten by `$(qt_feature ssl openssl_linked)` just below the `# Required for LibreSSL` comment. I presume LibreSSL still doesn't work without `openssl_linked`?

(Seems if I try to `Start a review` rather than `Add single comment` only I can see it...)

Reply:
Forgot to remove. Fixed